public/app/features/plugins/components/PluginsErrorsInfo.tsx

```diff
@@ -1,51 +1,65 @@
 import { css } from '@emotion/css';
 import React from 'react';
 
-import { GrafanaTheme2, PluginErrorCode, PluginSignatureStatus } from '@grafana/data';
+import { GrafanaTheme2, PluginErrorCode, PluginSignatureStatus, PluginType } from '@grafana/data';
 import { selectors } from '@grafana/e2e-selectors';
-import { HorizontalGroup, InfoBox, List, PluginSignatureBadge, useStyles2 } from '@grafana/ui';
+import { Alert, HorizontalGroup, Icon, List, PluginSignatureBadge, useStyles2 } from '@grafana/ui';
 
 import { useGetErrors, useFetchStatus } from '../admin/state/hooks';
 
-export function PluginsErrorsInfo() {
-  const errors = useGetErrors();
+type PluginsErrorInfoProps = {
+  filterByPluginType?: PluginType;
+};
+
+export function PluginsErrorsInfo({ filterByPluginType }: PluginsErrorInfoProps) {
+  let errors = useGetErrors();
```

Comment:
@jackw since you are now modifying this selector, you can also modify it to accept an optional parameter (plugin type) and let the selector do the filtering instead of the visual component.